src/components/Home/Home.js

```diff
@@ -78,7 +78,7 @@ class Home extends React.Component {
     }
   }
   render() {
-    const { connected, apps } = this.props
+    const { connected, apps, daoName } = this.props
```

Comment:
The DAO the user tries to access is already available as part of the the `locator` prop that already gets passed down, so we probably don't need another prop and state.

Reply:
Got it, will change it